src/category_theory/Groupoid.lean

```diff
@@ -8,7 +8,7 @@ import category_theory.groupoid category_theory.Cat
 /-!
 # Category of groupoids
 
-This file contains the definition of category `Groupoid` of all groupoids.
+This file contains the definition of the category `Groupoid` of all groupoids.
```

Comment:
Thanks @semorrison
It's difficult for me to use articles correctly, because we don't have them in Russian.